com.sejda.Desktop.metainfo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.sejda.Desktop</id>
+  <name>Sejda PDF Desktop</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>Sejda BV</developer_name>
+  <summary>Productive PDF software that you'll love to use</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">com.sejda.Desktop.desktop</launchable>
+  <url type="homepage">https://www.sejda.com/desktop</url>
+  <description>
```

Comment:
Add some more description please Add the features https://www.sejda.com/desktop as `li`